File: DevExpress-Demo/CS/GridDemo.Wpf/ModuleResources/DragDropViewModel.cs
using System.Collections.ObjectModel;
using System.Linq;
using DevExpress.Xpf.DemoBase.DataClasses;

namespace GridDemo {
    public class DragDropViewModel {
        public DragDropViewModel() {
            const int threshold = 9;
            ActiveEmployees = new ObservableCollection<Employee>(EmployeesData.DataSource.Skip(threshold));
            NewEmployees = new ObservableCollection<Employee>(EmployeesData.DataSource.Take(threshold));
        }

        public ObservableCollection<Employee> ActiveEmployees { get; set; }
        public ObservableCollection<Employee> NewEmployees { get; set; }
    }
}

File: DevExpress-Demo/VB/GridDemo.Wpf/ModuleResources/DragDropViewModel.vb
Imports System.Collections.ObjectModel
Imports System.Linq
Imports DevExpress.Xpf.DemoBase.DataClasses

Namespace GridDemo

    Public Class DragDropViewModel

        Public Sub New()
            Const threshold As Integer = 9
            ActiveEmployees = New ObservableCollection(Of Employee)(EmployeesData.DataSource.Skip(threshold))
            NewEmployees = New ObservableCollection(Of Employee)(EmployeesData.DataSource.Take(threshold))
        End Sub

        Public Property ActiveEmployees As ObservableCollection(Of Employee)

        Public Property NewEmployees As ObservableCollection(Of Employee)
    End Class
End Namespace
